lib/matplotlib/bezier.py

```diff
@@ -125,9 +125,10 @@ def find_bezier_t_intersecting_with_closedpath(bezier_point_at_t,
     end_inside = inside_closedpath(end)
 
     if not xor(start_inside, end_inside):
```

Comment:
Are start_inside and end_inside both Booleans?  If so, isn't this condition the same as (start_inside == end_inside)?